src/app/services/pass.service.ts

```diff
@@ -163,5 +164,33 @@ export class PassService {
     if (obj.facilityType === '' || !obj.facilityType) {
       throw ('You must provide a pass facility type');
     }
+    if (obj.token === '' || !obj.token) {
+      throw ('You must provide a token');
+    }
+    if (obj.commit === '' || !obj.commit) {
+      throw ('You must provide the commit attribute');
+    }
+  }
+
+  // Hold pass logic
+  async holdPass(postObj) {
+    postObj['commit'] = false;
+
+    // TODO: Validate we have all required attrs
+
+    try {
+      return await this.apiService.post('pass', postObj);
+    } catch (err) {
+      this.loggerService.debug(`${err}`);
+      throw err;
+    }
+  }
+
+  getDecodedToken(token: string): any {
+    try {
+      return jwtDecode(token);
+    } catch (Error) {
```

Comment:
This should be lowercase `error` not the `Error` class

Reply:
[a525da5](https://github.com/bcgov/parks-reso-public/pull/379/commits/a525da548aab47695ecafbdc621346deb5f26e1b)